src/metax_api/models/catalog_record_v2.py

```diff
@@ -126,6 +126,8 @@ def _post_create_operations(self, pid_type=None):
             else:
                 self.use_doi_for_published = False
 
+            self._handle_issued_date()
```

Comment:
This should be done in pre_create_operations for all datasets. Now it is actually not saved for datasets that are published straight away.

Reply:
Usch, you're right, same mistake again.. :E